babel/support.py

```diff
@@ -94,7 +94,7 @@ def datetime(
         >>> from babel.dates import get_timezone
         >>> fmt = Format('en_US', tzinfo=get_timezone('US/Eastern'))
         >>> fmt.datetime(datetime(2007, 4, 1, 15, 30))
-        u'Apr 1, 2007, 11:30:00\u202fAM'
+        'Apr 1, 2007, 11:30:00\\u202fAM'
```

Comment:
Why does a single slash turn into two here? (And in other places in the diff.)

Reply:
I explained in the PR, it's because this is a string within a string (a docstring).

> I also noticed some literals in docstrings were not escaped correctly (you need to escape the backslash unless it's a r""" docstring, otherwise the docstring will contain the actual character instead of the literal)